geomstats/geometry/dirichlet_distributions.py

```diff
@@ -516,10 +676,72 @@ def geodesic(self, initial_point, end_point=None,
             if initial_tangent_vec is not None:
                 raise ValueError('Cannot specify both an end point '
                                  'and an initial tangent vector.')
-            path = self._geodesic_bvp(initial_point, end_point, n_steps)
+            path = self._geodesic_bvp(initial_point, end_point, n_steps,
+                                      jacobian=jacobian, max_time=max_time)
 
         if initial_tangent_vec is not None:
             path = self._geodesic_ivp(
                 initial_point, initial_tangent_vec, n_steps)
 
         return path
+
+    def squared_dist(self, point_a, point_b, n_steps=N_STEPS, jacobian=False,
```

Comment:
Maybe we can add a `**kwargs` in the signature of squared_dist and propagate it to the log in the base class Riemannian metric so that you don't have to duplicate the code? It will be useful for other metrics where the log is an optimization problem with more key-word arguments.

Reply:
Good idea.